projects/components/src/description/description.component.scss

```diff
@@ -0,0 +1,23 @@
+:host {
```

Comment:
Is this doable without `:host`? we use it in some places, but it's a bad practice and we should try to get rid of it where possible (since the parent is generally responsible for styling the host, and will have greater precedence here)

Reply:
I'm try to do it without :host but for some reason style `min-width: 0 ` for `.description` not affected parent component. Originally this min-width: 0 added for fix such case as on screenshots.

Without min-width: 0 for host
![Screenshot from 2021-01-29 03-04-54](https://user-images.githubusercontent.com/76950513/106217923-f2daed80-61de-11eb-9d16-5adfd78dac1f.png)
Add min-width: 0
![Screenshot from 2021-01-29 03-04-43](https://user-images.githubusercontent.com/76950513/106217927-f3738400-61de-11eb-85c6-4eea38dae34e.png)

Reply:
There's usually alternatives, but always the easiest to find. Don't spend too much time on it - if we need to use :host so be it.